taxonomies/categories.txt

```diff
@@ -38319,8 +38540,10 @@ en:Koshihikari rices, Koshihikari
 de:Koshihikari Reis
 es:Arroces Koshihikari, Koshihikari
 fr:Riz Koshihikari, Koshihikari
+ja:コシヒカリ
 nl:Koshihikaririjsten, Koshihikaririjst
 wikidata:en:Q1204670
+# Should this really be part of en:Sushi rice?
```

Comment:
```suggestion
```

Reply:
As per my undertanding, Koshihikari is not only for sushi.
Then, I would suggest to remove "<en:Sushi rice"

Reply:
[Removed as so](https://github.com/Naruyoko/openfoodfacts-server/commit/4407609ad8dbbf1e59288c34c0d6ae2be656471e)